data-service/src/v1/dataservice-schema.graphql

```diff
@@ -0,0 +1,90 @@
+type Query {
```

Comment:
This isn't actually required, but keep it here for now as reference for the old API